### PR TITLE
Skip login if password is not set

### PR DESCRIPTION
**Pull request by blacksails**
This make the action able to run without providing a login

### CODE REVIEW

**Review by bgzzz — APPROVED** [review]
LGTM

File: src/docker.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import {info, warning, startGroup, endGroup} from '@actions/core';
+import {info, debug, warning, startGroup, endGroup} from '@actions/core';
 import {exec} from './exec';
 import {Inputs} from './inputs';
-import {setRegistry} from './state';
```
Comment (sorenmat) [inline src/docker.ts:4]: Why change this?

Comment (blacksails) [reply src/docker.ts:4]: I added an `outputs` module in the last PR. This change makes it clear that the set is in the state and not in the outputs. Eg. it is easy to understand what `state.setRegistry()` does without having to look at the imports.

Comment (blacksails) [reply src/docker.ts:4]: It's just to improve readability.

**Review by sorenmat — APPROVED** [review]

**Pull request merged**